test/workos/sso/sso_test.exs

```diff
@@ -30,7 +35,7 @@ defmodule WorkOS.SSOTest do
     setup do
       {:ok, url} =
         SSO.get_authorization_url(%{
-          domain: "test",
```

Comment:
It looks like this test block (`#get_authorization_url/2 with a domain`) was testing the `domain` parameter, so we can get rid of this entire test block since `domain` is no longer supported.